services/order.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime
+from typing import Any
+
+from django.db import transaction
+
+from db.models import Ticket, User, Order, MovieSession
+
+
+def create_order(tickets: list, username: str,
+                 date: datetime = None) -> Order:
```

Comment:
Consider adding type hints for the `tickets` parameter to specify the expected structure of the list elements, such as `list[dict[str, Any]]`, to improve code readability and maintainability.